Create makefile to install pre-built binaries, headers and new
pkg-config file
--- Makefile	1970-01-01 01:00:00.000000000 +0100
+++ Makefile	2016-11-10 17:31:08.686722661 +0000
@@ -0,0 +1,22 @@
+all:	libaltaircamlegacy.pc
+
+%.pc:	%.pc.in
+	sed -e "s!@LIBDIR@!$(LIBDIR)!g" \
+		-e "s!@VERSION@!$(VERSION)!g" < $< > $@
+
+install:
+	mkdir -p $(LIBDIR) $(INCDIR) $(PKGCONFDIR) $(UDEVDIR)
+	case $(DEB_BUILD_ARCH) in \
+	i386) \
+		cp $(SRCDIR)/linux/x86/libaltaircamlegacy*.so.$(VERSION) $(LIBDIR);; \
+	amd64) \
+		cp $(SRCDIR)/linux/x64/libaltaircamlegacy*.so.$(VERSION) $(LIBDIR);; \
+	aarch64) \
+		cp $(SRCDIR)/linux/arm64/libaltaircamlegacy*.so.$(VERSION) $(LIBDIR);; \
+	armhf) \
+		cp $(SRCDIR)/linux/armhf/libaltaircamlegacy*.so.$(VERSION) $(LIBDIR);; \
+	esac
+	ln -sf libaltaircamlegacy.so.$(VERSION) $(LIBDIR)/libaltaircamlegacy.so.$(MAJOR_REV)
+	ln -sf libaltaircamlegacy.so.$(VERSION) $(LIBDIR)/libaltaircamlegacy.so
+	cp $(SRCDIR)/inc/*.h $(INCDIR)
+	cp $(SRCDIR)/libaltaircamlegacy.pc $(PKGCONFDIR)
